src/test_vectors/zsa_issuance_auth_sig.rs

```diff
@@ -0,0 +1,279 @@
+// From https://github.com/QED-it/zcash-test-vectors/pull/12 (zsa_issuance_auth_sig)
+
```

Comment:
file name prefix `zsa_` looks ugly. Remove Prefix.

Reply:
Renamed file to `issuance_auth_sig`